backend/private-graph/graph/schema.resolvers.go

```diff
@@ -3632,6 +3632,30 @@ func (r *mutationResolver) UpdateEmailOptOut(ctx context.Context, token *string,
 	return true, nil
 }
 
+// EditService is the resolver for the editService field.
+func (r *mutationResolver) EditService(ctx context.Context, id int, projectID int, githubRepoPath *string) (*model.Service, error) {
+	project, err := r.isAdminInProject(ctx, projectID)
+	if err != nil {
+		return nil, err
+	}
+
+	serviceUpdates := map[string]interface{}{}
+	if githubRepoPath != nil {
+		serviceUpdates["GithubRepoPath"] = *githubRepoPath
+		serviceUpdates["Status"] = "healthy"
```

Comment:
This feels kind of misleading to me. Wouldn't `healthy` be the situation where we can actually verify data coming in from that service? In other words, just the process of linking a github repo doesn't really feel like it should be any different from the `created` status.

Reply:
Yeah, I don't feel too strongly here - I was thinking a connection would be deemed "healthy" until its not. Not sure if "healthy" is the right word, but for complexity not sure if we want to add logic that would come back and update the service when it starts processing to start